Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+
+try {
+    const input = fs.readFileSync(path.resolve(__dirname, "./input.txt"), "utf-8")
+                    .trim()
+                    .split('\n')
+                    .map(i => parseInt(i, 10));
+
+    // Part 1: Calculate the resulting frequency
+    const part1Result = input.reduce((a, b) => a + b, 0);
+    console.log("Part 1 - Resulting Frequency:", part1Result);
+
+    // Part 2: Find the first frequency reached twice
+    let sum = 0;
+    let set = new Set([0]);
```

Comment:
Why start with a set with one element, 0, instead of an empty set?

Reply:
The reason why we use 0 instead of an empty set is so that 0 is remembered as the starting frequency and won't be added into the set later and doesn't register it as a repeat and allows for the program to correctly detect when the frequency reaches 0 again.